lib/DataSift/ApiClient.php

```diff
@@ -117,6 +122,12 @@ static private function initialize($method, $ssl, $url, $headers, $params, $user
                 curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode($params));
                 break;
             }
+
+            case 'ingest': {
```

Comment:
These in the switch statement are to do with which type of REST call is being done, it should only contain, GET, POST, PUT, PATCH and DELETE. This ingest method is the same as POST